build/yaml/templates/versioning-steps.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - powershell: |
+      "Deployment Ring = $(DeploymentRing)";
+      $deploymentRing = "$(DeploymentRing)";
+      $dateStamp = (Get-Date -format "yyyyMMdd");
+      $commitHash = "$(Build.SourceVersion)".SubString(0,7);
+      $vs = $dateStamp + "." + $commitHash;
+
+      if ($deploymentRing -ne "stable") {
+        $vs = $deploymentRing + "." + $vs;
+        Write-Host "##vso[task.setvariable variable=VersionSuffix;]$vs";
```

Comment:
Does Write-Host have any significance here? I'm assuming it does since the other ones look to probably be logging but don't have it.

Reply:
Yeah it is different, it's how PS can set environment variables back into ADO.

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=classic%2Cpowershell#set-a-job-scoped-variable-from-a-script-1